### PR TITLE
Fix Extend the data model documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/extend_data_model.rst [hunk 1]
```diff
@@ -244,5 +245,53 @@ And finally in ``geoportal/<package>_geoportal/__init__.py`` replace ``config.sc
...
+
+   docker-compose exec geoportal python3
+
+Execute the following code:
+
+.. code:: python
+
+   from c2c.template.config import config
+
+   config.init('config.yaml')
```
Comment (arnaud-morvan) [inline doc/integrator/extend_data_model.rst:272]: May be `config.init('/etc/geomapfish/config.yaml')`, but you should now better than me

File: doc/integrator/extend_data_model.rst [hunk 2]
```diff
@@ -244,5 +245,53 @@ And finally in ``geoportal/<package>_geoportal/__init__.py`` replace ``config.sc
...
+
+   from sqlalchemy.schema import CreateTable
+   from <package>_geoportal.models import UserDetail, Title
+
+   print(CreateTable(UserDetail.__table__))
```
Comment (arnaud-morvan) [inline doc/integrator/extend_data_model.rst:277]: No need to initialize dbsession first ?

File: doc/integrator/extend_data_model.rst (hunk 1 above)
Comment (sbrunner) [reply doc/integrator/extend_data_model.rst:272]: +1 Thanks :-)

File: doc/integrator/extend_data_model.rst (hunk 2 above)
Comment (sbrunner) [reply doc/integrator/extend_data_model.rst:277]: No it's not needed :-)

**Pull request merged**